packages/asset-swapper/src/swap_quoter.ts

```diff
@@ -144,7 +148,12 @@ export class SwapQuoter {
      *
      * @return  An instance of SwapQuoter
      */
-    constructor(supportedProvider: SupportedProvider, orderbook: Orderbook, options: Partial<SwapQuoterOpts> = {}) {
+    constructor(
+        supportedProvider: SupportedProvider,
+        orderbook: Orderbook,
+        options: Partial<SwapQuoterOpts> = {},
+        quoteRequestor?: QuoteRequestor,
```

Comment:
Why not just add `quoteRequestor` to `SwapQuoterOpts`?

Reply:
Addressed in https://github.com/0xProject/0x-monorepo/pull/2541/commits/7096a70bc422bffecf7476110024fbb8cd78885c

Reply:
Addressed in https://github.com/0xProject/0x-monorepo/pull/2541/commits/63bfd233108f3fb9a918d722f2e4f0461e838b37